libraries/AdaptiveExpressions/BuiltinFunctions/Sqrt.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+
+namespace AdaptiveExpressions.BuiltinFunctions
+{
+    /// <summary>
+    /// Returns the logarithm of a specified number.
+    /// </summary>
+    internal class Sqrt : ExpressionEvaluator
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Sqrt"/> class.
+        /// </summary>
+        public Sqrt()
+            : base(ExpressionType.Sqrt, Evaluator(), ReturnType.Number, FunctionUtils.ValidateUnaryNumber)
+        {
+        }
+
+        private static EvaluateExpressionDelegate Evaluator()
+        {
+            return FunctionUtils.ApplyWithError(
+                        args =>
+                        {
+                            string error = null;
+                            object result = null;
+                            var originalNumber = Convert.ToDouble(args[0], CultureInfo.InvariantCulture);
+                            if (originalNumber < 0)
+                            {
+                                error = "Do not support square root extraction of negative numbers currently.";
```

Comment:
Remove "currently" from the error message.

Reply:
Ok. The orginal reason for 'current' is that currently, SDK does not support the square root of negative number in Real number scope. But in the future, we may consider supporting Complex number, after that the square root of the negative number is possible.
Anyway, remove the `currently` is acceptable.